pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -347,6 +348,7 @@ private static void start() {
         get(API.STUDY_DETAIL, new GetStudyDetailRoute(), responseSerializer);
         get(API.STUDY_PASSWORD_POLICY, new GetStudyPasswordPolicyRoute(), responseSerializer);
         get(API.STUDY_LANGUAGES, new ListStudyLanguagesRoute(), responseSerializer);
+        get(API.DISPLAY_LANGUAGE_POPUP, new GetShouldDisplayLanguageChangePopupRoute(), responseSerializer);
```

Comment:
Adding a new route for just one property seems like overkill. How about we extend the [GetStudyDetail API](https://pepper-dev.datadonationplatform.org/docs#operation/getStudyDetail) instead? It's a public route, and adding this "display language popup" setting there shouldn't be a problem.

Reply:
Good idea!